blackbox/docs/configuration.txt

```diff
@@ -1888,6 +1888,52 @@ Javascript Language
   Setting to enable the Javascript language. As The Javascript language is an
   experimental feature and is not securely sandboxed its disabled by default.
 
+Authentication
+==============
+
+.. _host_based_auth:
+
+Host Based Authentication
+-------------------------
+
+.. NOTE::
+
+   Host Based Authentication is a CrateDB Enterprise Edition feature.
+   This Authentication only concerns connections over the ``PostgreSQL
+   Wire Protocol``.
+
+**auth.host_based**
+  | *Runtime:*     ``no``
+  | *Examples:*
+  |  - ``{user:crate, address: 172.16.0.0/16}``
+  |  - ``{method: trust}``
+  |  - ``{user:crate, address: 172.16.0.0/16, method:trust}``
+
+  The host based configuration is a list of predicates that users can specify
+  to restrict access to CrateDB.
+
+  If ``auth.host_based`` is not configured, the host based authentication
+  is disabled. If the setting ``auth.host_based`` is present and the
+  configurations list does not contain any entry, then no one is allowed access.
+
+  The meaning of the fields is as follows:
```

Comment:
I think this needs to be described more accurate.
matching username and IP does not automatically mean access to crate, but only that the client is allowed to authenticate in a certain way (it is independent from trust authentication)